src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_pending_resumption_handler.cc

```diff
@@ -0,0 +1,351 @@
+#include "rc_rpc_plugin/rc_pending_resumption_handler.h"
+#include <sstream>
+#include "rc_rpc_plugin/rc_helpers.h"
+#include "rc_rpc_plugin/rc_module_constants.h"
+
+namespace rc_rpc_plugin {
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "RemoteControlModule")
+
+RCPendingResumptionHandler::RCPendingResumptionHandler(
+    application_manager::ApplicationManager& application_manager,
+    InteriorDataCache& interior_data_cache)
+    : ExtensionPendingResumptionHandler(application_manager)
```

Comment:
@jacobkeeler Make it sense to make proposed changes in the separate PR? Because these changes will affect a lot of files and Unit tests, that are not related to this PR. Or you would like to see these changes in this PR?

Reply:
Moved to #3475